cognite/client/data_classes/simulators/simulators.py

```diff
@@ -0,0 +1,517 @@
+    for monitoring and managing the interactions between CDF and external simulators, ensuring proper data flow and integration.
+    This is the read/response format of the simulator integration.
+    Args:
+        external_id (str): External id of the simulator integration
+        simulator_external_id (str): External id of the associated simulator
+        heartbeat (int): The interval in seconds between the last heartbeat and the current time
+        data_set_id (int): The id of the dataset associated with the simulator integration
+        connector_version (str): The version of the connector
+        license_status (str | None): The status of the license
+        simulator_version (str | None): The version of the simulator
+        license_last_checked_time (int | None): The time when the license was last checked
+        connector_status (str | None): The status of the connector
+        connector_status_updated_time (int | None): The time when the connector status was last updated
+    """
+
+    def __init__(
+        self,
+        external_id: str,
+        simulator_external_id: str,
+        heartbeat: int,
+        data_set_id: int,
+        connector_version: str,
+        license_status: str | None = None,
+        simulator_version: str | None = None,
+        license_last_checked_time: int | None = None,
+        connector_status: str | None = None,
+        connector_status_updated_time: int | None = None,
+    ) -> None:
+        self.external_id = external_id
+        self.simulator_external_id = simulator_external_id
+        self.heartbeat = heartbeat
+        self.data_set_id = data_set_id
+        self.connector_version = connector_version
+        self.license_status = license_status
+        self.simulator_version = simulator_version
+        self.license_last_checked_time = license_last_checked_time
+        self.connector_status = connector_status
+        self.connector_status_updated_time = connector_status_updated_time
+
+    @classmethod
+    def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> Self:
+        return cls(
+            external_id=resource["externalId"],
+            simulator_external_id=resource["simulatorExternalId"],
+            heartbeat=resource["heartbeat"],
+            data_set_id=resource["dataSetId"],
+            connector_version=resource["connectorVersion"],
+            license_status=resource.get("licenseStatus"),
+            simulator_version=resource.get("simulatorVersion"),
+            license_last_checked_time=resource.get("licenseLastCheckedTime"),
+            connector_status=resource.get("connectorStatus"),
+            connector_status_updated_time=resource.get("connectorStatusUpdatedTime"),
+        )
+
+    def dump(self, camel_case: bool = True) -> dict[str, Any]:
+        return super().dump(camel_case=camel_case)
+
+
+class SimulatorIntegrationWrite(SimulatorIntegrationCore):
```

Comment:
Why are we adding a write-class if the API only exposes filter and delete endpoints?